### PR TITLE
Add a sample sheet into the saccharomyces data

### PR DESCRIPTION
**Pull request by pinin4fjords**

### CODE REVIEW

**Comment by maxulysse** [comment]
shouldn't this be in the rnaseq branch then?

**Comment by pinin4fjords** [comment]
No- it's for module-level tests (specifically summarizedexperiment)

File: README.md [hunk 1]
```diff
@@ -537,6 +537,7 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
     - plastid
       - 'ddAraThal4.HiFi.reads.fasta': test dataset for plastid reads for Arabidopsis thaliana
   - saccharomyces_cerevisiae
+    - samplesheet.csv: sample sheet as used by the nf-core/rnaseq test profile
```
Comment (maxulysse) [inline README.md:540]: can you update the comment?

Comment (pinin4fjords) [reply README.md:540]: This is there to highlight where the data comes from

**Review by maxulysse — APPROVED** [review]

**Pull request merged**